Color.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>

```

Comment:
The code in this file can still be modularized into different files. 
An improvement could be by separating test code and production code into separate .h/.c files. By doing so, it will be helpful during future maintenance and readability.